sql/src/main/java/io/crate/execution/engine/collect/stats/JobsLogService.java

```diff
@@ -159,8 +159,14 @@ static long clearInterval(TimeValue expiration) {
             return NoopLogSink.instance();
         } else if (expirationMillis > 0) {
             q = new ConcurrentLinkedDeque<>();
-            TimeExpiring lbTimeExpiring = new TimeExpiring(clearInterval(expiration));
-            ScheduledFuture<?> scheduledFuture = lbTimeExpiring.registerTruncateTask(q, scheduler, expiration);
+            long delay = 0L;
```

Comment:
could this be a constant?

Reply:
Could be, but in this case I thought it's more readable to keep the value close to the use-site as well. Since it's single use and there is no cost associated with having a primitive 0L.